hadoop-hdds/docs/content/feature/OM-HA.md

```diff
@@ -125,7 +125,23 @@ ozone om [global options (optional)] --bootstrap --force
 
 Note that using the _force_ option during bootstrap could crash the OM process if it does not have updated configurations.
 
+## Automatic Snapshot Installation for Stale Ozone Managers
+
+Sometimes an OM follower node may be offline or fall so far behind the leader OM's log that it cannot catch up by replaying individual log entries.  The OM HA implementation includes an automatic snapshot installation and recovery process for such cases.
+
+How it works:
+
+1. Leader determines that the follower is too far behind.
+2. Leader notifies the follower to catch up via snapshot.
+3. The follower downloads and installs the latest snapshot from the leader.
```

Comment:
> ...  the latest snapshot from the leader.

Side note: we should support install snapshot from another follower in order to reduce the load of the leader.

Reply:
https://issues.apache.org/jira/browse/HDDS-13267